### PR TITLE
added planet category with its test class

### PR DESCRIPTION
**Pull request by Augilar**
I have created a new class for planets and added it into '__category map' and created a new test class for planets.

I am new to open source and happy to contribute 😁

### CODE REVIEW

File: qual_id/categories/planet.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class Planet(Category):
+  def get_values(self):
+    return [
+        'mercury',
+        'venus',
+        'earth',
+        'mars',
+        'jupiter',
```
Comment (gabrielbarker) [inline qual_id/categories/planet.py:11]: what about saturn? 😂

File: test/categories/test_planet.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import unittest
+from qual_id.categories.planet import Planet
+from test.utils.category_helper import CategoryHelper
+
+
+class TestColor(unittest.TestCase):
```
Comment (gabrielbarker) [inline test/categories/test_planet.py:6]:
```suggestion
class TestPlanet(unittest.TestCase):
```

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Hey @Augilar , thanks for taking the time to contribute! I've just spotted a couple of little tweaks, but other than that it looks great 🙂

File: qual_id/categories/planet.py (hunk 1 above)
Comment (Augilar) [reply qual_id/categories/planet.py:11]: How did I forget the best planet 😂

File: test/categories/test_planet.py (hunk 2 above)
Comment (Augilar) [reply test/categories/test_planet.py]: Sorry for that !!

**Comment by Augilar** [comment]
@gabrielbarker since language category has been merged do i have to change my pattern file to get my PR merged ?

**Comment by gabrielbarker** [comment]
Yeah @Augilar , since there's a conflict you'll have to resolve it by changing the file. You should be able to do it in the browser though I think, by clicking `Resolve conflicts`. Let me know if you need any help as it can be confusing 🙂

**Comment by Augilar** [comment]
@gabrielbarker Why are checks failing ? I have changed the file by clicking `Resolve conflicts` and then clicked `resolve` and then `merge`. Have I done something wrong ?

**Comment by gabrielbarker** [comment]
So if you look at the log for the test run [here](https://travis-ci.com/github/gabrielbarker/qual-id/jobs/394072341), you can see that the unit test for the pattern file has failed, and the error message is `"language": Language(),
SyntaxError: invalid syntax`. This tells us that the problem is to do with the __category_map because that's where that code is. If you then look at your changes [here](https://github.com/gabrielbarker/qual-id/pull/29/files) we can see that there is a missing comma after `Planet()` 🙂

The error messages aren't that great, I might look at improving them

File: qual_id/pattern.py [hunk 3]
```diff
@@ -39,6 +40,7 @@ class Pattern:
       "searchengine": SearchEngine(),
       "adjective": Adjective(),
       "city": City(),
+      "planet": Planet()
```
Comment (gabrielbarker) [inline qual_id/pattern.py:43]: Here's the offending line
```suggestion
      "planet": Planet(),
```

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

**Comment by Augilar** [comment]
Thanks for making me understand my mistake 😄.
I will improve and contribute more.

**Review by gabrielbarker — APPROVED** [review]

**Comment by gabrielbarker** [comment]
Thanks a lot @Augilar ! No one likes conflicts, but you handled them like a pro 💪

**Comment by Augilar** [comment]
Thanks a lot. I am happy that I am learnt how to handle conflicts from this contribution 😁

**Comment by Augilar** [comment]
Thanks a lot as I have learnt how to handle conflicts from this contribution 😁

**Pull request merged**